solver/llbsolver/file/user_linux.go

```diff
@@ -46,7 +46,8 @@ func readUser(chopt *pb.ChownOpt, mu, mg fileoptypes.Mount) (*copy.User, error)
 
 			ufile, err := os.Open(passwdPath)
 			if err != nil {
-				return nil, err
+				// Couldn't open the file. Considering this case as not finding the user in the file.
```

Comment:
Error should not be ignored on `!errors.Is(os.ErrNotExist) && !errors.Is(syscall.ENOTDIR)`

Reply:
Done